src/sidebar/test/group-fixtures.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+const Chance = require('chance');
+const chance = new Chance();
```

Comment:
I wasn't aware of Chance. This looks useful.

Reply:
The ungainly instantiation there (two lines instead of one) is due to a linting gripe I didn't feel like working around (if you care/are curious).